python/paddle/fluid/tests/book/high-level-api/fit_a_line/test_fit_a_line.py

```diff
@@ -57,22 +57,20 @@ def train(use_cuda, train_program, save_dirname):
         optimizer=fluid.optimizer.SGD(learning_rate=0.001))
 
     def event_handler(event):
-        if isinstance(event, fluid.EndEpochEvent):
-            test_metrics = trainer.test(
-                reader=test_reader, feed_order=['x', 'y'])
-            print test_metrics
-            '''
-            
-            ...
-            ['25.768919467926025']
-            ['15.343549569447836']
-            ...
-            
-            '''
-            if float(test_metrics[0]) < 20.0:
+        if isinstance(event, fluid.EndStepEvent):
+            if event.step == 10:
```

Comment:
Is this step for epoch? Or is it more like for iterations (different mini-batches across epochs)?

Reply:
Yeah, actually I was just confused about the naming. I was thinking that `event.step` should represent iterations. Probably, iterations should never counter back to 0. But in this case, we have `event.step` based on step_id. 

However, the step_id is from 0 to number of mini-batches. And after each epoch the step_id is reset to 0. 

So basically the point was that could be have a separate naming convention :)